README.md

```diff
@@ -539,6 +539,7 @@ A curated list of awesome Java frameworks, libraries and software. Inspired by o
 * [Play](http://www.playframework.com/) - Uses convention over configuration, hot code reloading and display of errors in the browser.
 * [Pippo](http://www.pippo.ro/) - A small, highly modularized Sinatra-like framework. 
 * [PrimeFaces](http://primefaces.org/) - JSF framework which has a free and a commercial version with support. Provides several frontend components.
+* [JavaServer Faces](https://javaserverfaces.java.net/) - Component-based UI framework; part of Java EE; many third-party component suites available with rich, powerful widgets.
```

Comment:
I know that you are professionally involved with JSF but we currently are leaning towards a more bleeding edge and community-driven list. You can see countless of discussions why we didn't add JSP, Ant, Hudson, etc. or are in progress to remove them.